centreon/tests/e2e/features/Dashboards/14-resource-table-widget-configuration.feature

```diff
@@ -26,17 +26,17 @@ Feature: Configuring resource table widget
     When the dashboard administrator user selects all the status and save changes
     Then only the unhandled resources are displayed in the resource table widget
 
-  @TEST_MON-35096
-  Scenario: Deleting a resource table widget
-    Given a dashboard featuring two resource table widgets
-    When the dashboard administrator user deletes one of the widgets
-    Then only the contents of the other widget are displayed
+  # @TEST_MON-35096
+  # Scenario: Deleting a resource table widget
+  #   Given a dashboard featuring two resource table widgets
+  #   When the dashboard administrator user deletes one of the widgets
+  #   Then only the contents of the other widget are displayed
 
-  @TEST_MON-35098
-  Scenario: Duplicating resource table widget
-    Given a dashboard having a configured resource table widget
-    When the dashboard administrator user duplicates the resource table widget
-    Then a second resource table widget is displayed on the dashboard having the same properties as the first widget
+  # @TEST_MON-35098
+  # Scenario: Duplicating resource table widget
+  #   Given a dashboard having a configured resource table widget
+  #   When the dashboard administrator user duplicates the resource table widget
+  #   Then a second resource table widget is displayed on the dashboard having the same properties as the first widget
```

Comment:
Why it's commented ?

Reply:
it's commented because it requires progress on the backend as soon as the matter is settled I'm going to uncomment them.

Reply:
Okay, but why aren't the changes being made at the same time? Because there are discrepancies between expectations and acceptances.  It's pretty risky.

Reply:
Because the backend changes are going to be made in another PR, so we're going to merge this PR and as soon as the backend is developed we're going to unmerge the gherkins.

Reply:
Ok